completion/nicematrix.cwl

```diff
@@ -0,0 +1,229 @@
+# nicematrix package
+# Matthew Bertucci 6/12/2021
+
+#include:amsmath
+#include:array
+#include:l3keys2e
+#include:pgfcore
+#include:xparse
+
+\NiceMatrixOptions{keyvals}
+
+\begin{NiceTabular}[keyvals]{preamble}[keyvals]#\tabular
```

Comment:
it seems not reasonable to have arguments ```[keyvals]{preamble}[keyvals]``` for the command completion
I would declare one variant for completer and maybe the complete one for syntax checking (#S)

Reply:
I've made the changes, however I was unsure if the `#S` lines should have `\tabular` and `\array` removed from the end, so I left them.